### PR TITLE
Untreated 1h actin

### PR DESCRIPTION
**Pull request by sbesson**
Reported by @francesw while reviewing #2, the experiments 7 and 8 both contain 4 channels although they are labelled as `actin_microtubules` only.

While for the testing work, we might just rename the structure via the UI/CLI, this PR modified the import files so that the images get correctly imported into `prod63`

### CODE REVIEW

**Comment by pwalczysko** [comment]
I have manually via the UI dragged all the zvi images (600 of them) into a new dataset called ``Untreated 1h Actin Vimentin``.  The remainder of the images, i.e. the lsm images (144 of them) are still in the original ``Untreated 1h Actin`` dataset. cc @francesw

File: scripts/generate_file_paths.py [hunk 1]
```diff
@@ -150,7 +150,11 @@ def get_treatment(imagename):
     if actin:
         datasetname += " Actin"
 
-    if "_vimentin_" in experiment:
+    # Although vimentin is missing from the filenames experimentA 7 and 8 are
+    # measuring acting and vimentin
```
Comment (pwalczysko) [inline scripts/generate_file_paths.py:154]: acting -> actin

**Comment by pwalczysko** [comment]
Comparing the manually-achieved result which is now on idr-testing with the diff of the script and the .tsv, I think this matches and would expect it to work.

I have to say I don't understand the diff of https://github.com/IDR/idr0050-springer-cytoskeletalsystems/pull/4/files#diff-7781b3cbef07bb62eaa7db96e06d4b89 - actually not clear why do we have the ``assays`` file and what purpose does it serve (of course just my ignorance, not to be taken as suggestion) ?

File: scripts/generate_file_paths.py [hunk 2]
```diff
@@ -151,7 +151,7 @@ def get_treatment(imagename):
         datasetname += " Actin"
 
     # Although vimentin is missing from the filenames experimentA 7 and 8 are
-    # measuring acting and vimentin
+    # measuring actin and vimentin
```
Comment (pwalczysko) [inline scripts/generate_file_paths.py]: +1

**Comment by sbesson** [comment]
Re the removal of the assays files, these are the author-submitted files that were originally committed in eb749af . Given that we have now decided to combine all the study into one experiment, the `experimentB/` folder is confusing. Instead of deleting these files, we could keep it as it is to retain the original metadata but move it under `experimentA`?

**Comment by pwalczysko** [comment]
>we could keep it as it is to retain the original metadata but move it under experimentA?

I think so. @francesw  ?

**Comment by francesw** [comment]
I agree with keeping it to retain the original metadata and moving it under Experiment A.

**Comment by sbesson** [comment]
:+1: done with https://github.com/IDR/idr0050-springer-cytoskeletalsystems/pull/4#event-2106421969

**Comment by pwalczysko** [comment]
Ready to merge FMPOV

**Pull request merged**